doc/classes/Object.xml

```diff
@@ -764,6 +764,23 @@
 				Emits the [signal property_list_changed] signal. This is mainly used to refresh the editor, so that the Inspector and editor plugins are properly updated.
 			</description>
 		</method>
+		<method name="property_can_revert" qualifiers="const">
+			<return type="bool" />
+			<param index="0" name="property" type="StringName" />
+			<description>
+				Returns [code]true[/code] if the given [param property] has a custom default value. Use [method property_get_revert] to get the [param property]'s default value.
+				[b]Note:[/b] The object must implement [method _property_can_revert], either in its script or in the engine code. If [method _property_can_revert] is not implemented, this method returns [code]false[/code].
+				[b]Note:[/b] This method returns consistently, regardless of the current value of the [param property].
+			</description>
+		</method>
+		<method name="property_get_revert" qualifiers="const">
+			<return type="Variant" />
+			<param index="0" name="property" type="StringName" />
+			<description>
+				Returns the custom default value of the given [param property]. Use [method property_can_revert] to check if the [param property] has a custom default value.
+				[b]Note:[/b] The object must implement [method _property_get_revert], either in its script or in the engine code. If [method _property_get_revert] is not implemented, this method returns [code]null[/code].
```

Comment:
I'm not sure if _"or in the engine code"_ is important information. To most users the way the engine does this internally doesn't change much, I feel. What changes? Does a property with no default custom value return `null`? Or, does it return default empty initialization (for example, an empty Array or Vector with all components set to 0)?

Reply:
The reason why I'm adding this is because some built-in classes use this feature, but most don't. So I think it's important to inform the user that in some cases this feature can be used with built-in classes, not just user-scripted classes. If I only mention that you need to implement it, and if it's not implemented then nothing happens, it would look like this is only for scripting.

Reply:
We could rephrase the whole note though, if you have a better idea.

Reply:
Something along the lines of:
```suggestion
				[b]Note:[/b] For the majority of properties (including built-in), this method returns [code]null[/code]. To customise their default value, override [method _property_get_revert].
```

Although the more I look at the implementation of these 4 methods the stranger they look.

Reply:
They are narrowly useful, so explaining them in a general sense is hard. And it doesn't help that the "can" method is not how we do it in other such cases. But it's too late to change now.

Reply:
@Mickeon I changed it a bit by shifting the focus instead of rewriting. Let me know if it works better now.

Reply:
I like the focus on the Inspector icon. Reading it makes the methods feel like they do have an editor-specific purpose (as they indeed do), similar to `notify_property_list_changed()`.